src/pages/content-edit/invalid-msg.js

```diff
@@ -0,0 +1,80 @@
+import m from "mithril";
+
+import css from "./invalid-msg.css";
+
+// A mosly-dumb controller is required here so we can retain some state 
```

Comment:
I mean I won't lie, mithril does make it a real pain in the ass. But it also looks nice, and I have to assume we're going to want an automatic dismiss mechanism of some kind. If we want auto-dismiss we're going to have to wire up the same dumb things to make it happen.